src/vmm_mad/remotes/lib/lxd/client.rb

```diff
@@ -130,7 +130,14 @@ def get_response(request, data)
 
         response.reading_body(@socket, request.response_body_permitted?) {}
 
-        response = JSON.parse(response.body)
+        begin
+            response = JSON.parse(response.body)
+        rescue StandardError => e
```

Comment:
Catch the exception you are interested in instead of match on error messages

begin
...
rescue JSON::ParserError
...
end